Docs/Citations.txt

```diff
@@ -1,193 +0,0 @@
-@article{marquet2020phage,
```

Comment:
@mult1fractal this file is removed in this PR
do manual merge so its not removed in the merge

Reply:
nvm its renamed to .bib my bad